### PR TITLE
Add workflow to lint react-app

### PR DESCRIPTION
**Pull request by TaylorFries**
## 🎯 Summary

  
[PIMS-621: ](https://citz-imb.atlassian.net/browse/PIMS-621)

Add linting workflow for react-app folder

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.

### CODE REVIEW

File: .github/workflows/react-app-lint.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        working-directory: ./react-app
+        run: npm i --only=dev
+
+      - name: Run ESLint
+        working-directory: ./react-app
+        run: npm run lint
+
+      - name: Run Prettier
+        working-directory: ./react-app
+        run: npm run check
```
Comment (dbarkowsky) [inline .github/workflows/react-app-lint.yml:32]:
```suggestion
        run: npm run check 
        
```
New line missing

File: .github/workflows/react-app-lint.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      # Could optimize this by only installing necessary dependencies
+      - name: Install Dependencies
+        working-directory: ./react-app
+        run: npm i --only=dev
```
Comment (dbarkowsky) [inline .github/workflows/react-app-lint.yml:24]: I always assumed this worked, but now I see this output:
image

Doesn't hurt to have it in, but maybe it's not doing what we thought.

**Review by dbarkowsky — APPROVED** [review]
Looks to be running properly. 👍

File: .github/workflows/react-app-lint.yml (hunk 2 above)
Comment (TaylorFries) [reply .github/workflows/react-app-lint.yml:24]: After some searching it seems like this flag stopped working in 2020, any updated command seems to not replicate what this did.

**Pull request merged**